doc/datavolume-annotations.md

```diff
@@ -0,0 +1,65 @@
+# Data Volume Annotations
+
+## Introduction
+
+CDI allows adding specific annotations to a DV or PVC to be passed to the Transfer Pods and control their behavior. Only specifically allowed annotations are passed, to prevent unexpected behavior due to missing testing coverage. However any needed annotation can be added to the allowed list.
```

Comment:
I assume the allowed list isn't controlled in the "user can change it once running CDI" way.
Not sure that the bit about the allowed list being changeable should be in this documentation.

Reply:
removed